File: problem-solving/beecrowd/1008-salario/src/main.c
#include <stdio.h>

int main(){
  int number, hours_worked;
  double wage_per_hour, salary;
  scanf("%d\n%d\n%lf", &number, & hours_worked, &wage_per_hour);
  salary = wage_per_hour * hours_worked;
  printf("NUMBER = %d\n", number);
  printf("SALARY = U$ %.2lf\n", salary);
}

File: problem-solving/beecrowd/1008-salario/src/main.rs
use std::io;

fn input() -> f64 {
    let mut buffer = String::new();
    io::stdin()
        .read_line(&mut buffer)
        .expect("Didn't read input.");

    let input : f64 = buffer
        .trim()
        .parse()
        .expect("Informed number is not a float f64.");
    return input;
}

fn main() {
    let number = input();
    let hours = input();
    let wage = input();
    let salary = hours * wage;
    println!("NUMBER = {}\nSALARY = U$ {:.2}", number as i64, salary);
}
